app/services/postcode_service.rb

```diff
@@ -20,6 +20,23 @@ def infer_la(postcode)
     end
   end
 
+  def infer_admin_district(postcode)
```

Comment:
Isn't this the same as the method above?

Reply:
Returns different value - I will refactor those two into one, to either return code and value or return postcode object